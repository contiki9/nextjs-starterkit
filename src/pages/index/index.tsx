import { useDisclosure } from '@chakra-ui/react'
import React from 'react'
import { Meta } from 'src/components/global'
import { Flex } from 'src/components/layout'
import { DefaultLayout } from 'src/components/page/_layout'
import { Button } from 'src/components/core'

export default function HomePage() {
  const { isOpen, onOpen, onClose } = useDisclosure()
  return (
    <>
      <Meta />
      <Flex justify="center" align="center" direction="column">
        <DefaultLayout>
          contents here
          <Button>Button</Button>
        </DefaultLayout>
      </Flex>
    </>
  )
}
