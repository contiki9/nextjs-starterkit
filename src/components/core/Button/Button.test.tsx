import React from 'react'

import ShallowRenderer from 'react-test-renderer/shallow'

import { Button } from './index'

const render = ShallowRenderer.createRenderer()
describe('<Button />', () => {
  it('初期表示', () => {
    render.render(<Button>Button</Button>)
    const container = render.getRenderOutput()
    expect(container).toMatchSnapshot()
  })
})
