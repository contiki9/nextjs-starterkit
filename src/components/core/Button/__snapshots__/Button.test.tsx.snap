// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> 初期表示 1`] = `
<Button
  color="primary.500"
  size="md"
  variant="solid"
>
  Button
</Button>
`;
